<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="2"
           failures="2"
           errors="0"
           skipped="0"
           time="0.306">
  <properties />
            <testcase name="tc_003_checkProceedToCheckoutButtonPresent" time="0.150" classname="org.web.automation.testcases.TC_006_ProductPopupWindowFunctionality">
                <failure type="java.lang.AssertionError"
                  message="expected [true] but found [false]"
              >
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
  org.testng.Assert.fail(Assert.java:99)
  org.testng.Assert.failNotEquals(Assert.java:1037)
  org.testng.Assert.assertEqualsImpl(Assert.java:140)
  org.testng.Assert.assertEquals(Assert.java:122)
  org.testng.Assert.assertEquals(Assert.java:819)
  org.testng.Assert.assertEquals(Assert.java:829)
  org.web.automation.utility.Validate.validateElementPresent(Validate.java:15)
  org.web.automation.pages.ProductAddedPopup.ProceedToCheckoutButtonPresent(ProductAddedPopup.java:86)
  org.web.automation.testcases.TC_006_ProductPopupWindowFunctionality.tc_003_checkProceedToCheckoutButtonPresent(TC_006_ProductPopupWindowFunctionality.java:25)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:564)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
  org.testng.TestRunner.privateRun(TestRunner.java:764)
  org.testng.TestRunner.run(TestRunner.java:585)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.runSuites(TestNG.java:1069)
  org.testng.TestNG.run(TestNG.java:1037)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
            <testcase name="tc_002_checkContinuebuttonPresent" time="0.156" classname="org.web.automation.testcases.TC_006_ProductPopupWindowFunctionality">
                <failure type="java.lang.AssertionError"
                  message="expected [true] but found [false]"
              >
<![CDATA[java.lang.AssertionError: expected [true] but found [false]
  org.testng.Assert.fail(Assert.java:99)
  org.testng.Assert.failNotEquals(Assert.java:1037)
  org.testng.Assert.assertEqualsImpl(Assert.java:140)
  org.testng.Assert.assertEquals(Assert.java:122)
  org.testng.Assert.assertEquals(Assert.java:819)
  org.testng.Assert.assertEquals(Assert.java:829)
  org.web.automation.utility.Validate.validateElementPresent(Validate.java:15)
  org.web.automation.pages.ProductAddedPopup.ContinueButtonPresent(ProductAddedPopup.java:74)
  org.web.automation.testcases.TC_006_ProductPopupWindowFunctionality.tc_002_checkContinuebuttonPresent(TC_006_ProductPopupWindowFunctionality.java:18)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
  java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.base/java.lang.reflect.Method.invoke(Method.java:564)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
  org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
  org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
  org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
  org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
  org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
  java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
  org.testng.TestRunner.privateRun(TestRunner.java:764)
  org.testng.TestRunner.run(TestRunner.java:585)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
  org.testng.SuiteRunner.run(SuiteRunner.java:286)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
  org.testng.TestNG.runSuites(TestNG.java:1069)
  org.testng.TestNG.run(TestNG.java:1037)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
      </failure>

          </testcase>
  
  
  
</testsuite>
